pkg/api/server/v1alpha2/result/result_test.go

```diff
@@ -81,23 +84,26 @@ func TestParseName(t *testing.T) {
 }
 
 func TestToStorage(t *testing.T) {
-	got, err := ToStorage("foo", "bar", &pb.Result{
-		Name: "foo/results/bar",
-		Id:   "a",
+	apiResult := &pb.Result{
```

Comment:
nit: this isn't being reused, so let's just leave it inlined like it was before.

Reply:
Great! Done